protocol/v2/ssv/validator/validator.go

```diff
@@ -205,6 +197,14 @@ func validateMessage(share spectypes.Share, msg *queue.SSVMessage) error {
 	return nil
 }
 
+func (v *Validator) loggerForDuty(logger *zap.Logger, role spectypes.RunnerRole, slot phase0.Slot) *zap.Logger {
```

Comment:
`loggerForDuty` and `trySetDutyID` seem very similar, can we merge them into one or maybe reuse one of them in the other? It's kind of out of the scope of the PR but since you're removing some `trySetDutyID` calls, it might be nice to fix it

Reply:
idk, but Ive refactored the logger functions to support a chained pattern with the `with*` prefix naming convention. So I also removed the unnecessary function `trySetDutyID`